js/header.js

```diff
@@ -70,7 +52,7 @@
         if (!offCanvasToggle) {
           return;
         } else {
-          offCanvasToggle.addEventListener("click", handleOffCanvas);
+          offCanvasToggle.addEventListener("click", handleOpenOffCanvas);
           window.addEventListener("keyup", function (e) {
             if (e.code === "Escape") {
               handleCloseOffCanvas();
```

Comment:
Realistically, in terms of closing modal behaviour, we also need to listen for click event outside of `offCanvas` when offCanvas is open.

Not sure if can be included in logic line 63 so listening to click on `offCanvasClose` and outside `offCanvas` (when open), but something like `offCanvas.contains(e.target)` inside click eventListener should be enough.

Reply:
Clicking outside of offCanvas will now close the offCanvas. 4b7a963

Reply:
That looks like it does the job nicely, thanks!